easyrest/views/auth_controler.py

```diff
@@ -0,0 +1,77 @@
+"""Controler which defines auth behavior
+"""
+import datetime as dt
+
+from pyramid.view import view_config
+from pyramid.authentication import AuthTicket
+from pyramid.httpexceptions import HTTPForbidden
+
+from ..scripts.json_helpers import wrap
+from ..models.user import User
+from ..models.token import Token
+from ..auth import restrict_access, remember, forget
+
+
+@view_config(route_name='login', renderer='json', request_method='GET')
+def login_get(request):
```

Comment:
I understand it's temporary, but why do you need it now? especially users tokens?